src/@next/components/molecules/ResetPasswordForm/ResetPasswordForm.tsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+
+import { Button } from "@components/atoms";
+import { TextField } from "../TextField";
```

Comment:
I guess it should be `@components/molecules`

Reply:
In this case we must stick to following way of importing things.
If we developing component `@components/molecules/B` and want to import `@components/molecules/A`then we use `import { A } from "../A"`. Using `import { A } from "@components/molecules"` would make our app crash. Its issue when importing from the same component type directory ;)

Reply:
Ah, ok